### DISCUSSION TITLE
Build Error

### DISCUSSION THREAD

**Discussion by EinWilderRex** [ocornut/imgui#5554] [New Users Build/Link/Run issues ONLY!]
Hey, use ImGui for now. I create a window and close with functions etc.

ImGui_ImplDX9_CreateDeviceObjects();
ImGui_ImplDX9_InvalidateDeviceObjects();
ImGui_ImplDX9_NewFrame();
ImGui_ImplWin32_NewFrame();
ImGui_ImplDX9_Shutdown();
ImGui_ImplWin32_Shutdown();

But get an error. Can someone help me with this?
Microsoft error: https://docs.microsoft.com/de-de/cpp/error-messages/tool-errors/linker-tools-error-lnk2001?f1url=%3FappId%3DDev16IDEF1%26l%3DDE-DE%26k%3Dk(LNK2001)%26rd%3Dtrue&view=msvc-170

![1](https://user-images.githubusercontent.com/61328908/183321051-3eb0ff34-d445-4daa-bb29-29886d8a3936.PNG)

**Comment by rokups** [comment]
You need to compile following files in your project as well:
- imgui.cpp
- imgui_demo.cpp
- imgui_draw.cpp
- imgui_tables.cpp
- imgui_widgets.cpp
- imgui_impl_win32.cpp
- imgui_impl_dx9.cpp
